app/src/main/java/com/github/studenttimetracker/database/DatabaseHelper.java

```diff
@@ -16,7 +16,7 @@
 
 public class DatabaseHelper extends OrmLiteSqliteOpenHelper {
 
-    private static final String DATABASE_NAME = "timetracker.db";
+    public static final String DATABASE_NAME = "timetracker.db";
```

Comment:
robienie jakiegoś pola public jest antypatternem. W sensie u nas to wszystko jedno ale lepiej zostawić private i dać adnotacje @getter nad tym polem albo ustawić customowego gettera